.github/workflows/remove-wheels.yml

```diff
@@ -0,0 +1,80 @@
+name: Remove old wheels
+
+on:
+  # Run daily at 1:23 UTC
+  schedule:
+  - cron:  '23 1 * * *'
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+# Needed for micromamba pickup
+defaults:
+  run:
+    shell: bash -l {0}
+
+env:
+  N_LATEST_UPLOADS: 5
+  ANACONDA_USER: "scientific-python-nightly-wheels"
+
+jobs:
+  remove:
+
+    runs-on: ubuntu-latest
+    # Set required workflow secrets in the environment for additional security
+    # https://github.com/scientific-python/upload-nightly-action/settings/environments
+    environment:
+      name: remove-old-wheels
```

Comment:
@stefanv as I don't have org level admin permissions can I get you to verify that the url in the comment (https://github.com/scientific-python/upload-nightly-action/settings/environments) correctly dumps you into the Environments list view and that someone has also setup the `remove-old-wheels` environment with the secret?

Reply:
@stefanv @jarrodmillman ping on this as you're the org owners (and so I think(?) are the only ones with the required level of permissions to do this).